sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 1.15.0-beta.1 (Unreleased)
 
+### Added
```

Comment:
Given [these guidelines](https://azure.github.io/azure-sdk/policies_releases.html#change-logs) I think this should be "### Features Added".

Reply:
BTW: I fixed the site so the ToC and Guidelines menu use the same data source so easier to get to policies. 🎉